.github/workflows/test.yaml

```diff
@@ -137,18 +137,36 @@ jobs:
           - python-version: '3.11'
 
           # Pre-release Python 3.13 with default settings.
-          # For now, no dependencies as cffi doesn't work on 3.13.
           - python-version: '3.13-dev'
-            tox-env: 'nodeps-withcov-posix'
-            job-name: 'nodeps-3.13'
-            # FIXME:https://github.com/twisted/twisted/issues/12060
-            # Also add twisted.python
-            # FIXME:https://github.com/twisted/twisted/issues/12061
-            # Add full twisted.test
-            trial-target: 'twisted.conch twisted.cred twisted.internet twisted.logger twisted.mail twisted.names twisted.pair twisted.persisted twisted.protocols twisted.runner twisted.spread twisted.test.test_defer twisted.trial twisted.web twisted.words'
-            # Disable concurrency for now, since we get occasional segfaults and
-            # this helps debug them:
-            trial-args: '--reactor=default'
+            tox-env: 'alldeps-withcov-posix'
+            job-name: 'alldeps-3.13'
+
+          # Pre-release Python 3.13 on macOS with default settings.
+          - python-version: '3.13-dev'
+            runs-on: 'macos-12'
+            tox-env: 'macos-withcov-alldeps'
+            job-name: 'macos-alldeps-3.13'
```

Comment:
Maybe it's better to move them after `job-name: 'macos-12-default-tests'` 


For macos we run limtied tests... and I guess that once py3.13 is released we will stop running tests on macoS with py3.12